using-timescaledb/continuous-aggregates.md

```diff
@@ -181,9 +181,7 @@ SELECT drop_chunks(INTERVAL '30 days', 'device_readings')
 You can read more about data retention with continuous aggregates in
 the [*Data retention*][sec-data-retention] section.
 
->:TIP: Most times the continuous aggregate view will be updated by the background job;
-  however, if you would like to run it yourself, you may use the
-  [`REFRESH MATERIALIZED VIEW` command][api-refresh-continuous-aggs].
+>:TIP: Most times the continuous aggregate view will be updated by the background job; however, if you would like verify that the view is working correctly, you may use the [`REFRESH MATERIALIZED VIEW` command][api-refresh-continuous-aggs].
```

Comment:
This is a bit misleading IMO. The background job to refresh the view can be failing even if a manual refresh updates the view. So, I don't think a manual refresh is good  indication of the automation working.

Reply:
Removed the tip.